addons/website/tests/test_fuzzy.py

```diff
@@ -225,7 +225,7 @@ def test_06_case_insensitive_results(self):
 
     def test_07_no_fuzzy_for_mostly_number(self):
         """ Ensures exact match is used when search contains mostly numbers. """
-        self._create_page('Product P7935432254U7 page', 'Product P7935432254U7', '/numberpage')
+        self._create_page('Product P7935432254U7 page', 'Product P7935432254U7 kangaroo shoes', '/numberpage')
```

Comment:
Commit message:
```
[FIX] website: adapt fuzzy test

Changes done in this task introduce new data that involve the word
"product" the fuzzy test is based on.

This commit adapts the test so that its results are based on words
that are less likely to appear in default data.

task-2406626
```
?

Reply:
It looks more descriptive and easy to read the changes on the commit 👍 

Done 👌